src/app/public/styles/ag-grid-styles.scss

```diff
@@ -0,0 +1,124 @@
+@import '~@skyux/theme/scss/mixins';
+
+/* base grid theme property variables */
+$odd-row-background-color: #FFFFFF !default;
+$background-color: $sky-color-gray-01 !default;
+$border-color: $sky-border-color-neutral-medium !default;
+$row-height: 38px !default;
+$header-height: 37px !default;
+$font-family: "Blackbaud Sans" !default;
+$header-background-color: #FFFFFF !default;
+$font-size: 15px !default;
+$row-border-width: 1px !default;
+$secondary-font-size: 15px !default;
+$secondary-font-weight: 400 !default;
+$secondary-foreground-color: $sky-text-color-deemphasized !default;
+$primary-color: $sky-highlight-color-info !default;
+$selected-color: #F1EEF6 !default;
+$hover-color: #E9E4F1 !default;
+
+/* other variables */
+$standard-border: 1px solid $border-color !default;
+
+@import "~ag-grid-community/src/styles/ag-grid.scss";
+@import "~ag-grid-community/src/styles/ag-theme-balham/sass/ag-theme-balham.scss";
+
+ag-grid-angular {
+  width: 100%;
+  height: 100%;
+}
+.ag-theme-balham {
+  .ag-root {
+    border: none;
+    border-left: $standard-border;
+    border-right: $standard-border;
+  }
+
+  .ag-header-cell::after {
```

Comment:
This is probably a good opportunity for a comment to explain what you're hiding.